<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>三维脑影像压缩系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/ui.qrc">
    <normaloff>:/ui/window.ico</normaloff>:/ui/window.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QPushButton" name="pushButton_compress">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始压缩</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_mode">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>ABR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CQP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CRF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lossless</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>QP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>码率(bps)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>CRF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>最大参考帧数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>关键帧间隔</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>B帧数量</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_preset">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <item>
     <property name="text">
      <string>ultrafast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>superfast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>veryfast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>faster</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>slow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>slower</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>veryslow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>placebo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>码率模式</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>编码预设</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>优化目标</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_optimize">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>psnr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ssim</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>grain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zerolatency</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>fastdecode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>animation</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox_SAO">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>样本自适应偏移</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_LF">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>340</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>去块滤波器</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_SS">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>370</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>强帧内平滑</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>881</width>
      <height>251</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>879</width>
       <height>249</height>
      </rect>
     </property>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>881</width>
        <height>251</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextInteractionFlag::TextSelectableByMouse</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="widget_orign" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QVTKOpenGLNativeWidget" name="widget_restore" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>321</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_crf">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>51.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>23.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_qp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>309</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>51</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_bps">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="singleStep">
     <number>500</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_mrf">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_kf">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_bf">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
    </property>
    <property name="maximum">
     <number>999999999</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openfile_action"/>
   <addaction name="separator"/>
   <addaction name="batchcompress_action"/>
   <addaction name="separator"/>
   <addaction name="restore_action"/>
   <addaction name="separator"/>
  </widget>
  <action name="openfile_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="toolTip">
    <string>打开.nii文件并显示</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="batchcompress_action">
   <property name="text">
    <string>批量压缩</string>
   </property>
   <property name="toolTip">
    <string>批量压缩.nii文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="restore_action">
   <property name="text">
    <string>批量重建</string>
   </property>
   <property name="toolTip">
    <string>批量将.h265压缩文件重建为.nii文件</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/ui.qrc"/>
 </resources>
 <connections/>
</ui>
